### ISSUE TITLE
Settings Profiles Don't Appear on Ubuntu 18.04

### ISSUE THREAD

**Issue by CTAnthny** [BasioMeusPuga/Lector#90]
![screenshot from 2019-02-17](https://user-images.githubusercontent.com/15718285/52919576-c4092c80-32d1-11e9-852e-4a9de148b315.png)

**Comment by BasioMeusPuga** [comment]
Have you tried clicking on the first button in the toolbar? The one that looks like an A?

**Comment by CTAnthny** [comment]
Yes, it is only showing background color setting :frowning_face: 
![screenshot from 2019-02-18](https://user-images.githubusercontent.com/15718285/52962476-54e01680-336b-11e9-8a5e-c761a7bce706.png)

**Comment by BasioMeusPuga** [comment]
That's because PDFs are rendered as images. You'll see the usual image profiles in books containing text. 

Help me understand what you'd want to happen. Do you want different colors for the page and the text for pdf files? I'm not sure that's something I'd want. I can invert the page colors, but that's about the extent of it.

**Comment by CTAnthny** [comment]
Ah.. I'd just like to be able to dark theme or custom theme style the text and background color for PDFs as well. Okular does that through the color inversion like you mentioned, but it sounds like the limitation is based on it being a PDF?
![screenshot from 2019-02-18](https://user-images.githubusercontent.com/15718285/52976141-4a874200-3396-11e9-90ed-5cdd8e9bc73a.png)

**Comment by BasioMeusPuga** [comment]
Yup. As far as I know, PDFs have a fixed layout.
I've committed something that will allow for the inversion of page colors. So there's that.
